modules/install-ibm-cloud-configuring-the-public-subnet.adoc

```diff
@@ -0,0 +1,160 @@
+// This is included in the following assemblies:
+//
+// installing_ibm_cloud/install-ibm-cloud-installing-on-ibm-cloud.adoc
+
+[id="configuring-the-public-subnet_{context}"]
+= Configuring the public subnet
+
+All of the {product-title} cluster nodes must be on the public subnet. IBM Cloud&#174; does not provide a DHCP server on the subnet. Set it up separately on the provisioner node. This document describes configuring Dnsmasq on the public subnet.
+
+.Procedure
+
+. Ensure `dnsmasq` is running:
++
+[source,terminal]
+----
+$ systemctl status dnsmasq
+----
++
+If `dnsmasq` isn't running, start it and enable it:
++
+[source,terminal]
+----
+$ sudo systemctl start dnsmasq
+----
++
```

Comment:
No need to do any of this, we havn't configure dnsmasq yet

Reply:
Deferred to later in the process. I also restored installing dnsmasq to the beginning of this procedure. Our modules SHOULD stand alone, and not necessarily depend on preceding modules.